docs/apv.md

```diff
@@ -557,9 +557,9 @@ Onderstaande regels tellen alleen op Narcopolis
 3. Bij overtreding van lid 1 of lid 2 zal een straf worden gegeven van de 1e categorie.
 4. Er mag op binnenkomende vluchten en schepen geschoten worden, die niet afkomstig zijn van de koninklijke marechaussee.
 
-### Artikel N-2 - Landen van helikopers/vliegtuigen
+### Artikel N-2 - Inladen van helikopers/vliegtuigen
 
-1. Het is verplicht om te landen op een helipad, een landingsbaan of in een straal van 50 meter van een helipad/landingsbaan.
+1. Wanneer jij drugs wilt inladen in een vliegtuig of helikopter moet het vliegtuig of de helikopter binnen een straal van 50 meter van een helipad of landingsbaan staan.
 2. De sea sparrow en dodo mag wel op open water rondom het eiland geland worden.
```

Comment:
Is dit nu nog relevant gezien de naam van het artikel is veranderd?

Reply:
Zonder dit lid te veranderen, kan het nog zo zijn dat er onduidelijkheden zijn die zijn hierbij weggenomen.

Reply:
welke onduidelijkheid? dit gaat over landen terwijl het artikel over inladen gaat. dit is een en al onduidelijkheid tbh